### PR TITLE
Devx fileapetitionsentancecase

### PR DESCRIPTION
**Pull request by alsmith1**
changes to title capitalization.

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]

File: web-client/integration-tests/journey/taxpayerCreatesNewCaseTestAllOptions.js [hunk 1]
```diff
@@ -363,7 +363,7 @@ export default (test, fakeFile, overrides = {}) => {
     await test.runSequence('updateStartCaseFormValueSequence', {
       key: 'estateType',
       value:
-        'Estate without an Executor/Personal Representative/Fiduciary/etc.',
+        'Estate without an executor/personal representative/Fiduciary/etc.',
```
Comment (kkoskelin) [inline web-client/integration-tests/journey/taxpayerCreatesNewCaseTestAllOptions.js]: *`fiduciary` ?

File: web-client/src/views/DraftDocuments/DraftDocuments.jsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ export const DraftDocuments = connect(
             <thead>
               <tr>
                 <th>Date</th>
-                <th>Filings and Proceedings</th>
+                <th>Filings and proceedings</th>
                 <th>Created By</th>
```
Comment (kkoskelin) [inline web-client/src/views/DraftDocuments/DraftDocuments.jsx]: *`Created by` ?

**Review by kkoskelin — APPROVED** [review]
A few observations about changes that may need to be fixed; if I'm wrong about them, go ahead and merge. :)

File: web-client/src/views/DraftDocuments/DraftDocuments.jsx (hunk 2 above)
Comment (alsmith1) [reply web-client/src/views/DraftDocuments/DraftDocuments.jsx]: created by was not one of the slides natalie set up but yes it should be updated as well

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**